### ISSUE TITLE
Can't import folder/index.mjs with `import './folder'`

### ISSUE THREAD

**Issue by mAAdhaTTah** [standard-things/esm#65]
You can `const module = require('./folder')` when the folder has an `index.js` but `import module from './folder';` produces this error:

```js
Error [ERR_MISSING_MODULE]: Cannot find module ../function
    at /private/tmp/test-esm/main.mjs
    at Object.<anonymous> (/private/tmp/test-esm/main.mjs)
    at Module._compile (module.js:573:30)
```

[Reproducible testcase](https://github.com/mAAdhaTTah/import-folder-esm)

Is this intended?

**Comment by jdalton** [comment]
Thanks @mAAdhaTTah!

Patched https://github.com/standard-things/esm/commit/f0f26049cffca2fed1126623aab36844a825cd93.

**Comment by davidbonnet** [comment]
Can't wait to upgrade from `0.2.2`. 🤗

**Comment by jdalton** [comment]
Fixed by v0.6.0. Please note I've [changed some usage instructions](https://github.com/standard-things/esm#getting-started).
